server/src/test/scala/com/metamx/tranquility/server/ServerConfigTest.scala

```diff
@@ -44,6 +44,5 @@ class ServerMainTest extends FunSuite with ShouldMatchers
     fooConfig.fireDepartment.getTuningConfig.getWindowPeriod should be(new Period("PT30S"))
     fooConfig.config.zookeeperConnect should be("zk.example.com")
     fooConfig.config.taskPartitions should be(3)
-    fooConfig.config.druidBeamConfig.firehoseGracePeriod should be(new Period("PT1S"))
```

Comment:
That's a very good question, I didn't explicitly remove it. Something weird with the merge, I'll look into it.